samples/client/petstore/spring-stubs/src/main/java/org/openapitools/api/PetApi.java

```diff
@@ -64,6 +64,7 @@ default ResponseEntity<Void> addPet(
     ) {
         return new ResponseEntity<>(HttpStatus.NOT_IMPLEMENTED);
 
+
```

Comment:
cant' figure out where this new line come from 🥲

Reply:
Me neither